string destination # 'warehouse' or 'garbage'
---
